assets/src/edit-story/components/header/buttons/switchToDraft.js

```diff
@@ -43,10 +51,20 @@ function SwitchToDraft() {
     saveStory,
   ]);
 
+  const label = __('Switch to Draft', 'web-stories');
   return (
-    <Outline onClick={handleUnPublish} isDisabled={isSaving || isUploading}>
-      {__('Switch to Draft', 'web-stories')}
-    </Outline>
+    <Tooltip title={label} placement={TOOLTIP_PLACEMENT.BOTTOM} hasTail={true}>
+      <Button
+        variant={BUTTON_VARIANTS.SQUARE}
+        type={BUTTON_TYPES.TERTIARY}
+        size={BUTTON_SIZES.SMALL}
+        onClick={handleUnPublish}
+        disabled={isSaving || isUploading}
+        aria-label={label}
+      >
+        <Icons.SwitchToDraft />
+      </Button>
+    </Tooltip>
```

Comment:
I'm afraid that someone might have changed the designs while you're working on them.

[Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=327%3A10229):
![Screen Shot 2021-02-12 at 00 08 08](https://user-images.githubusercontent.com/637548/107732511-8aa00780-6cc6-11eb-8fb3-5da23324b4f3.png)

Checking with @samitron7 [here](https://xwp.slack.com/archives/C01ESBB8ZN1/p1613106676206900).

Reply:
Turns out the design is not changing. Lucky you 😁  🍾 .

Reply:
And then again maybe it is, but stick to the current path for now.

(see above thread for more)